helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1263,6 +1263,7 @@ instrumentation:
   #   - repository: Specifies the Docker image repository.
   #   - tag: Indicates the Docker image tag.
   #   - env: (Optional) Allows you to add any additional environment variables.
+  # Auto-instrumentation Libraries (Start)
```

Comment:
What's the added value of this comment? I see the comment block's first line is almost identical, so I'm wondering why we need this line as well.

Reply:
Good question. 

We use the text tokens `# Auto-instrumentation Libraries (Start)` and `# Auto-instrumentation Libraries (End)` to identify the exact section of the values.yaml file that should be updated.

With our current CI/CD tools, if we didn’t specify a section to modify, the entire file would be rewritten (including formatting and comments) resulting in undesirable and systematic changes.

That said, I believe we could improve/replace this process with Renovate CI/CD tools and some additional developer work. This isn’t a priority right now, but it’s something we may consider in the future. For now, we’re using a minimal solution that ensures only the necessary updates are made while preserving the rest of the file.